test/pop_protocol_test.erl

```diff
@@ -93,7 +93,7 @@ basic_test() ->
     FoldFn = fun(Block, PD) -> 
```

Comment:
Maybe rename FoldFn to AccumBlockchain or something and move it to a utility module?

Reply:
or I guess just to blocktree

Reply:
👍